src/tests/ftest/launch.py

```diff
@@ -47,7 +47,7 @@
 DEFAULT_DAOS_TEST_LOG_DIR = os.path.join(os.sep, "var", "tmp", "daos_testing")
 DEFAULT_DAOS_TEST_USER_DIR = os.path.join(os.sep, "var", "tmp", "daos_testing", "user")
 DEFAULT_DAOS_TEST_SHARED_DIR = os.path.expanduser(os.path.join("~", "daos_test"))
-DEFAULT_LOGS_THRESHOLD = "2G"
+DEFAULT_LOGS_THRESHOLD = "2.1G"
```

Comment:
The problem in the ticket is the rotated server log (`DAOS_Management_daos_server_1.log.83131.old`) is just a bit larger than the 2G threshold.  There is a race condition where this can happen and rather than complicate the log rotation code to ensure we never exceed the rotation size, it was deemed simpler to add a little wiggle room in the log threshold check.

Reply:
AFAIK we have only seen this in release/2.2 so far - at least in terms of daily runs - so to confirm this is working we may need to repeat running the `daos_test/suite.py` test or run this in the release/2.2 branch.  In either case a detailed examination of the `artifact/Functional%20Hardware%20Medium/launch/functional_hardware_medium/*-daos_test-suite-launch.log` will be required.